timeline-chart/README.md

```diff
@@ -0,0 +1,14 @@
+## Description
+
+A time graph / gantt chart library for large data (e.g. traces).
+
+## Additional Information
+
+- [Documentation](https://github.com/theia-ide/timeline-chart/blob/master/doc/documentation.md): Detailed description of the timeline chart library and it's components.
+- [Theia Trace Extension](https://github.com/theia-ide/theia-trace-extension): An reference application that uses timeline the chart library.
```

Comment:
An > A
timeline the chart > the timeline chart

Reply:
Done